### PR TITLE
ANA-1754: ensure report currency added on lookthrough computations

### PR DESCRIPTION
**Pull request by fbn-seamus**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: examples/use-cases/ibor/Maintain a fund in multiple currencies and share classes.ipynb [hunk 1]
```diff
@@ -128,7 +128,7 @@
      "output_type": "stream",
```
Comment (rizwansaeed) [inline examples/use-cases/ibor/Maintain a fund in multiple currencies and share classes.ipynb:128]: <p>should the <code>USTreasury_6.875_2025</code> value be changing? (I thought they were going to stay the same?) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/568/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (rizwansaeed) [inline examples/use-cases/ibor/Maintain a fund in multiple currencies and share classes.ipynb:128]: <p>general comment (I havent commented on all of them), but it looks like the PV numbers are changing </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/568/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='118'/>

File: examples/use-cases/valuation/Look-through valuation.ipynb [hunk 2]
```diff
@@ -162,20 +162,20 @@
        "    <tr>\n",
```
Comment (rizwansaeed) [inline examples/use-cases/valuation/Look-through valuation.ipynb:162]: <p>seem to be a different set of instruments, i wonder if the luids are being reused. the pv numbers are the same so i think its ok</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/568/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

File: examples/use-cases/ibor/Maintain a fund in multiple currencies and share classes.ipynb (hunk 1 above)
Comment (fbn-seamus) [reply examples/use-cases/ibor/Maintain a fund in multiple currencies and share classes.ipynb:128]: yep, PV numbers changing are expected, they actual change without my change. I think they were picking up some dud data in the environment that was previously being run against. This change doesn't actually change the numbers on top of that.

Comment (fbn-seamus) [reply examples/use-cases/ibor/Maintain a fund in multiple currencies and share classes.ipynb:128]: <p>yup, that number seems to have been picking up some dud data from the run that it was recorded on in the previous change.</p>

[the remark this answers is not in the document]
Comment (fbn-seamus) [reply examples/use-cases/ibor/Maintain a fund in multiple currencies and share classes.ipynb:128]: <p>yep, it's the US treasury one being reflected here.</p>

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**